zeppelin-web/src/app/notebook/notebook.css

```diff
@@ -221,3 +232,41 @@
   text-decoration: none;
   cursor: default;
 }
+
+.userlist {
+  width: 320px;
```

Comment:
@ravicodder It's my personal opinion but, isn't it too long comparing to above `search for users` box ? :)